README.md

```diff
@@ -116,17 +115,17 @@ Responds with one of:
       }
       ```
 
-### /$gpc.ack
+### POST /$gpc.ack
 
-The ack endpoint is the final endpoint to call once you are happy that the EHR record you have received is acceptable.
+This endpoint finalizes the EHR transfer process.
 If you do not call this endpoint after receiving an EHR from the migratestructuredrecord enpoint, then you risk the losing practise triggering off the manual postal transfer.
 
-The following is required to call this endpoint:-
+Required fields:
 
-- CONVERSATION_ID: The id associated with the patient transfer request.
-- CONFIRMATION_RESPONSE: you can provide the following status:-
-    - ACCEPTED: This will tell the sending incumbent that you are happy with the received EHR.
-    - FAILED_TO_INTEGRATE: You have encountered a problem integrating the record into your system. This will alert the sender to an error and trigger off the postal process.
+- CONVERSATION_ID: ID from the initial request.
+- CONFIRMATION_RESPONSE: Status of the EHR integration.
+    - ACCEPTED: EHR integration successful.
+    - FAILED_TO_INTEGRATE: Error encountered; triggers postal process.
```

Comment:
We should be specific about how to provide these fields, I also don't think the names match the actual expected values. 😬